codecarbon/cli/main.py

```diff
@@ -25,12 +25,13 @@
 from codecarbon.emissions_tracker import EmissionsTracker
 
 AUTH_CLIENT_ID = os.environ.get(
-    "AUTH_CLIENT_ID", "pkqh9CiOkp4MkPqRqM_k8Xc3mwBRpojS3RayIk1i5Pg"
+    "AUTH_CLIENT_ID",
+    "jsUPWIcUECQFE_ouanUuVhXx52TTjEVcVNNtNGeyAtU",
```

Comment:
This auth token expose no secret and can be in the source ?

Reply:
it's the oauth2 client id, not secret, it' public